00 Intro/01 JSX/src/components/panel-component/PanelComponent.tsx

```diff
@@ -0,0 +1,21 @@
+import * as React from 'react';
+
+interface Props {
+  title: string;
+  children?: React.ReactNode | React.ReactNode[];
```

Comment:
Do you pass children via props, instead of extends PanelComponent from React.Component because you don't want stateful component? Or what is the reason?